plugins/coverbrowser.koplugin/main.lua

```diff
@@ -434,7 +428,6 @@ function CoverBrowser:addToMainMenu(menu_items)
                         keep_menu_open = true,
                         callback = function()
                             local ConfirmBox = require("ui/widget/confirmbox")
-                            UIManager:close(self.file_dialog)
```

Comment:
Why these are not needed ? The file_dialog auto-closes when button pressed ? Or you want to keep them open?

Reply:
They're the main menu items. What is file_dialog?

Reply:
Dunno, don't want to have to look at the code and where these comes from :) So, you're telling me that there is self.file_dialog, and they probably come from some old copy&paste ? Trusting you.

Reply:
It's the upper menu, not popup dialog.

Reply:
So, you're telling me that there is **no** self.file_dialog, and they probably come from some old copy&paste ?

Reply:
I don't know, erroneous lines from 2017.